Formula/vis.rb

```diff
@@ -0,0 +1,54 @@
+class Vis < Formula
+  desc "Vim-like text editor"
+  homepage "https://github.com/martanne/vis"
+  url "https://github.com/martanne/vis/archive/v0.4.tar.gz"
+  sha256 "f11ba41cfb86dd39475960abfd12469de4da0ccfdb941f1d7680d89d987694c5"
+  head "https://github.com/martanne/vis.git"
+
+  depends_on "lua" => :build
+  depends_on "libtermkey"
+
+  resource "lpeg" do
+    url "https://luarocks.org/manifests/gvvaughan/lpeg-1.0.1-1.src.rock", :using => :nounzip
+    sha256 "149be31e0155c4694f77ea7264d9b398dd134eca0d00ff03358d91a6cfb2ea9d"
+  end
+
+  def script; <<-EOS.undent
```

Comment:
Just inline this inside `def install` since it's only being used the once.

```ruby
(bin/"vise").write <<~EOS
   blah
EOS
```
etc.